codecs/vp8_packet.go

```diff
@@ -79,6 +83,11 @@ type VP8Packet struct {
 func (p *VP8Packet) Unmarshal(packet *rtp.Packet) ([]byte, error) {
 	payload := packet.Payload
 
+	// If lower than the required header size, malformed header
+	if len(payload) < 4 {
+		return nil, fmt.Errorf("empty payload")
```

Comment:
I would delete the comment and put it in the errorf `Payload is not large enough to container header`